src/v/rpc/test/rpc_integration_fixture.h

```diff
@@ -97,7 +98,9 @@ class rpc_base_integration_fixture {
     explicit rpc_base_integration_fixture(uint16_t port)
       : _listen_address("127.0.0.1", port)
       , _ssg(ss::create_smp_service_group({5000}).get0()) {
-        _sg = ss::create_scheduling_group("rpc scheduling group", 200).get0();
+        auto suffix = random_generators::gen_alphanum_string(10);
+        _sg = ss::create_scheduling_group("rpc scheduling group" + suffix, 200)
+                .get0();
```

Comment:
does it work to create a static scheduling group and reuse that for each of the tests? the reason that is probably relevant here is that there is a limit to the number of scheduling groups which is like 32 or 64 or something.

Reply:
Not sure are you saying this limit spans tests? The problem here is once you create 2 instances of this fixture you will observe the assertion due to the name not being unique.